apply-personal-number.md

```diff
@@ -0,0 +1,31 @@
+# How to apply personal number
+## Step 1:
+##### At first check your map, search a nearest “tax agency”, “tax office” or ”Skattekontoret” (Swedish).
+
+## Step 2：
+##### Bring your Passport, Work permit card; Marriage certificate & legalized translation if not single, Kid's birth certificate & legalized translation if have kids to Skattekontoret. Take a ticket and wait.
+
+## Step 3:
+##### The officer will check your files and you can fill the website on the computer they will find for you. Fill it step by step.
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/1.png)
```

Comment:
Should use relative path here. Refer [here](https://github.com/Orientalent/guidelines/commit/cfd435fb6741b427ad0f17d37ebe4e738268ea61#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)